doc/gazebo_simulation/gazebo_simulation.rst

```diff
@@ -0,0 +1,351 @@
+Gazebo Simulation Integration
+=============================
+
+`Gazebo <http://gazebosim.org/>`_ is the most popular robotics simulator in the ROS ecosystem, so is naturally a good fit to integrate with MoveIt.
+
+The `MoveIt Setup Assistant <../setup_assistant/setup_assistant_tutorial.html>`_ helps setup your robot to work with Gazebo, but there are still additional steps required to successfully run MoveIt in Gazebo.
+
+----------------------------
+After MoveIt Setup Assistant
+----------------------------
+This tutorial assumes that the robot is set up with `MoveIt Setup Assistant <../setup_assistant/setup_assistant_tutorial.html>`_,
+so it is crucial to follow that document first. To the best of our knowledge, official FRANKA EMIKA Panda repository doesn't particularly consider Gazebo simulation
+such that the necessary components to properly simulate the robot in Gazebo are missing with respect to :code:`urdf` and :code:`xacro` files. This is a rare incident, since most other robots
+have those components out of the box. *If you have a custom robot, which already works well in Gazebo, you can skip the steps until Step-6.* Fortunately, there is already a good solution offered in `the blog post <https://erdalpekel.de/?p=55>`_ to this problem. For the sake of completion though,
+the procedure outlined in there will be repeated (with improvements) in here as well for preparing the robot for Gazebo simulation.
+**Note that these steps assume that you have cloned franka_ros repository from the source**:
+
+1. Fix the robot to the world coordinate system
+2. Add damping to the joint specifications
+3. Add inertia matrices and masses to the links
+4. Add friction and colorize the links
+5. Configure gazebo_ros_control, transmissions and actuators
+6. Adjust auto-generated ros_controllers.yaml
+7. Adjust auto-generated ros_controllers.launch
+
+
+1. Fix the robot to the world coordinate system
+-----------------------------------------------
+Open the :code:`franka_description/robots/panda_arm_hand_urdf.xacro` file and change the line 5 with:
+
+.. code-block:: xml
+
+    <xacro:panda_arm xyz="0 0 0" rpy="0 0 0" connected_to="world"/>
+
+It alone doesn't fix the problem, since now we need to provide a link with name :code:`world`. Open the 
+:code:`franka_description/robots/panda_arm.xacro` file and add:
+
+.. code-block:: xml
+
+    <link name="world" />
```

Comment:
I would suggest to add the `<link name="world" />` inside `franka_description/robots/panda_arm_hand_urdf.xacro` and not in `franka_description/robots/panda_arm.xacro`. I think is more clean since `panda_arm_hand_urdf.xacro` call the macro specifying `connect_to="world"` and therefore I expect the world to be defined outside the macro.